hypha/apply/api/v1/tests/test_serializers.py

```diff
@@ -26,3 +31,45 @@ def test_handles_negative_reviews(self):
         self.assertEqual(data['recommendation'], {'value': 0, 'display': 'No'})
         self.assertEqual(len(data['assigned']), 1)
         self.assertEqual(len(data['reviews']), 1)
+
+
+class TestDeliverableSerializer(TestCase):
+    def test_id_is_required(self):
+        serializer = DeliverableSerializer(data={'quantity': 1})
+        self.assertFalse(serializer.is_valid(), serializer.error_messages)
```

Comment:
You can also add another assert, assertEqual, to actually validate the error message that is being raised.